plugins/cost-insights/src/utils/graphs.ts

```diff
@@ -23,29 +23,30 @@ import {
   lengthyCurrencyFormatter,
 } from './formatters';
 
-export function formatGraphValue(
-  value: number,
-  _index: number,
-  format?: string,
-) {
-  if (format === 'number') {
-    return value.toLocaleString();
-  }
+export const formatGraphValue =
+  (baseCurrency: string) =>
+  (value: number, _index: number, format?: string) => {
+    if (format === 'number') {
+      return value.toLocaleString();
+    }
 
-  if (value < 1) {
-    return lengthyCurrencyFormatter.format(value);
-  }
+    if (value < 1) {
+      return lengthyCurrencyFormatter(baseCurrency).format(value);
+    }
 
-  return currencyFormatter.format(value);
-}
+    return currencyFormatter(baseCurrency).format(value);
+  };
 
 export const overviewGraphTickFormatter = (millis: string | number) =>
   typeof millis === 'number' ? dateFormatter.format(millis) : millis;
 
-export const tooltipItemOf = (payload: Payload<string, string>) => {
+export const tooltipItemOf = (
+  baseCurrency: string,
+  payload: Payload<string, string>,
+) => {
   const value =
-    typeof payload.value === 'number'
-      ? currencyFormatter.format(payload.value)
+    payload.value && !isNaN(Number(payload.value))
```

Comment:
Cause `payload.value === 'number'` never can't be a number here. 
If you open the code in Intellij IDEA, it even shows that right away.

![Screenshot 2022-11-17 at 11 55 30](https://user-images.githubusercontent.com/2265094/202428185-873446e7-17f5-4258-843c-4990b67d0eac.png)

And if to have a look at the code, the input parameter is payload: `Payload<string, string>`, not payload: `Payload<string | number, string>`. And changing that is not possible because `Payload<string, string>` comes from `recharts`